apps/main-landing/src/app/creators/donate/hooks/use-sender.ts

```diff
@@ -70,7 +70,7 @@ export const useSender = ({ walletClient, publicClient }: Properties) => {
         walletClient,
       });
 
-      if (isNativeTokenAddress(sendPayload.tokenAddress)) {
+      if (isNativeTokenAddress(sendPayload.tokenAddress as `0x${string}`)) {
```

Comment:
we can remove type casting here and make an update in the `donate/context.tsx` (https://github.com/idriss-xyz/core/pull/184/files#r1874046247)

Reply:
removed